app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -501,17 +492,6 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
                                     webView.evaluateJavascript(script) {
                                         Log.d(TAG, "Callback $it")
                                     }
-
-                                    getAndSetStatusBarNavigationBarColors()
```

Comment:
Since we support back ~ 6 months worth of core versions I say we leave this.  Just add a comment that after 2022.6 releases we can remove it.